doc/tutorials/transformer/sequence_classification.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>see my note in filling mask notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>